client/asset/eth/eth.go

```diff
@@ -3828,18 +3839,15 @@ func (w *assetWallet) balanceWithTxPool() (*Balance, error) {
 		}
 	}
 
+	confirmed, err := getBal()
+	if err != nil {
+		return nil, fmt.Errorf("balance error: %v", err)
+	}
```

Comment:
This change was recommended by @martonp at https://github.com/decred/dcrdex/pull/2211#discussion_r1140800213. I thought I had already done this, but apparently not.